jobs/test_e2e_errand/templates/bin/run.erb

```diff
@@ -15,40 +16,63 @@ export PATH="/var/vcap/packages/java8/bin:$PATH"
 
 [ ! -e /home/vcap/.sincedb ] || rm /home/vcap/.sincedb
 
-
-echo '' ; echo '==> Purging existing test data...'
-
-curl -s -X DELETE '<%= p('test_e2e_errand.elasticsearch_hostport') %>/_all/sample-logs--repo-commits' ; echo ''
-
+export QUERY=$(cat <<EOF
+{ "query": {
+    "filtered" : {
+        "query" : {
+            "term" : { "@source.host":"sample-logs--repo-commits" }
+        },
+        "filter" : {
+          "bool" : {
+            "must" : {
+                "range" : {
+                    "@timestamp" : {
+                        "from" : "$(date --iso-8601=ns)",
```

Comment:
I'm not sure why `@timestamp` should be necessary here. Without the `DELETE`, there will be multiple records from each test run since the field it is extracted from the commit date (in which case I don't understand how this was working and always returning 32 records). With the `DELETE`, the filter shouldn't be necessary because all records should already be from the specific test's run. Am I missing something?